test/operator/installation.go

```diff
@@ -12,11 +12,12 @@ import (
 )
 
 const (
-	localManifestsDir          = "config/deploy/kubernetes/"
-	kubernetesCsiManifest      = "kubernetes-csi.yaml"
-	kubernetesOperatorManifest = "kubernetes.yaml"
-	openshiftCsiManifest       = "openshift-csi.yaml"
-	openshiftOperatorManifest  = "openshift.yaml"
+	localKubernetesManifestsDir = "config/deploy/kubernetes/"
+	localOpenshiftManifestsDir  = "config/deploy/openshift/"
```

Comment:
could be split into common path constant `config/deploy` and the different platforms `kubernetes`/`openshift`